src/Arr.php

```diff
@@ -726,4 +726,57 @@ public function isEmpty(): bool
     {
         return $this->length() === 0;
     }
+
+    /**
+     * Returns a string representing the elements of the array
+     * @return string the string representation
+     */
+    public function toLocaleString($locale = 'en_US.utf8', $timezone = 'UTC'): string
```

Comment:
Do we want to add type declarations here to ensure that we are always passing in strings for these values since we have left it open for users to pass in anything else into this method

Reply:
I added param type hints with default values